Automattic/wp-calypso
# Login Assistance: Revert the "show migrate CTA when login fails" experiment

Resolves https://github.com/Automattic/wp-calypso/issues/93074

## Proposed Changes

* Revert all the code related to the experiment.

## Why are these changes being made?


* Based on the related experiments, it was decided that this is not something that we want to pursue. More context here: pdxWSz-1G5-p2#comment-587

## Testing Instructions

* Go to http://calypso.localhost:3000/log-in.
* Test the login form.
* Make sure the error message is displayed when you fail to log in.
* Make sure the form works correctly when you log in with the correct credentials.

## Pre-merge Checklist
